client/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ConwaysGameEngine } from '@monarchwadia/conways-game-engine';
+import './App.css';
+
+
+
+function App() {
+
+
+
+const grid = new ConwaysGameEngine();
+const random = (Math.random()*10);
+
+let randomspot = grid.draw(Math.round(random), Math.round(random))
+
```

Comment:
if you want a random number I would recommend making a function and returning a random number everytime you call it... for example: 
```javascript
function getRandomNum() {
  return Math.round(Math.random()*10)
}

let randomSpot = grid.draw(getRandomNum(), getRandomNum());
```